docs/pages/example/fallback-image.js

```diff
@@ -1,6 +1,6 @@
 /*---
 title: Use a fallback image
-description: Fallback to another image when a requested image is not available.
+description: Use a [coalesce](https://docs.mapbox.com/mapbox-gl-js/style-spec/expressions/#coalesce) expression to fall back to another image when a requested image is not available.
```

Comment:
I wonder if part of the confusion is unfamiliarity with the term "fallback" -- what do you think of using a more generic word/phrase to describe what is happening?

```suggestion
description: Use a [coalesce](https://docs.mapbox.com/mapbox-gl-js/style-spec/expressions/#coalesce) expression to display another image when a requested image is not available.
```